### PR TITLE
Adding more github resources

### PR DESCRIPTION
**Pull request by Shruti9520**

### CODE REVIEW

File: _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md~ [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
```
Comment (abbycabs) [inline _articles/get-your-project-online/project-set-up-for-collaboration-with-github.md~]: Hey @Shruti9520! It looks like this file was commit by mistake. Can you remove this from your PR?

**Review by abbycabs — CHANGES_REQUESTED** [review]
This looks amazing except for the one extra file. Maybe we should add *~ to .gitignore?

Thanks for your work here @Shruti9520!

**Comment by Shruti9520** [comment]
@acabunoc I have deleted that extra file.

**Comment by Shruti9520** [comment]
@acabunoc Do I need to make other changes in this?

**Review by abbycabs — APPROVED** [review]

**Comment by abbycabs** [comment]
Thanks @Shruti9520!

**Pull request merged**